integration_tests/suite/test_transfers.py

```diff
@@ -538,20 +534,32 @@ def assert_transfer_is_blind_transferred(
 
     def assert_transfer_is_abandoned(
         self,
-        transfer_id,
-        transferred_channel_id,
-        initiator_channel_id,
-        recipient_channel_id,
-        events,
+        transfer_id: str,
+        transferred_channel_id: str,
+        initiator_channel_id: str,
+        recipient_channel_id: str,
+        events: BusMessageAccumulator,
     ):
-        transfer_bridge = self.ari.bridges.get(bridgeId=transfer_id)
-        assert_that(
-            transfer_bridge.json,
-            has_entry(
-                'channels',
-                contains_inanyorder(initiator_channel_id, recipient_channel_id),
-            ),
-        )
+        def receive_transfer_events():
+            assert_that(
+                events.accumulate(with_headers=True),
+                has_items(
```

Comment:
Because now, an abandoned transfer does not mean that it has ended yet, as it may be in an intermediary `abandoned` state.
If the expectation is that the transfer is abandoned and the ended, this should require two seperate checks.